app/mailers/notification_mailer.rb

```diff
@@ -411,8 +411,8 @@ def default_url_options
     super.merge(host: provider_account&.external_admin_domain)
   end
 
-  def t_subject(key, options = {})
-    I18n.t(key, { scope: 'mailers.notification_mailer.subject', raise: true }.merge(options))
+  def t_subject(key, **options)
+    I18n.t(key, **{ scope: 'mailers.notification_mailer.subject', raise: true }.merge(options))
```

Comment:
I think this may work but has to be tested
```suggestion
    I18n.t(key, scope: 'mailers.notification_mailer.subject', raise: true, **options)
```